extensions/amp-video/0.1/test-e2e/test-amp-video.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describes.endtoend(
+  'amp-video autoplay with control',
+  {
+    testUrl: 'http://localhost:8000/test/fixtures/e2e/amp-video/autoplay.html',
+    environments: 'amp4ads-preset',
+  },
+  env => {
+    let controller;
+
+    beforeEach(async () => {
+      controller = env.controller;
+    });
+
+    function isPaused(elem) {
+      return controller.getElementProperty(elem, 'paused');
+    }
+
+    it('play control', async () => {
```

Comment:
Nit: I would modify the naming of this test to be something that indicates what it tests for, e.g. it('should autoplay the video and display controls').

Reply:
Great suggestion. Updated.